geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/netty/Coder.java

```diff
@@ -471,4 +472,101 @@ public static int narrowLongToInt(long toBeNarrowed) {
     }
     return doubleBytes;
   }
+
+  /**
+   * Takes the given "value" and computes the sequence of ASCII digits it represents,
+   * appending them to the given "buf".
+   * This code was adapted from the openjdk Long.java getChars methods.
+   */
+  public static void appendAsciiDigitsToByteBuf(long value, ByteBuf buf) {
+    if (value < 0) {
+      buf.writeByte('-');
```

Comment:
This char can be replaced with the `bMINUS` constant.

Reply:
done